classes/module/Module.php

```diff
@@ -2036,6 +2036,11 @@ public function displayWarning($warning)
         return $output;
     }
 
+    /**
+    * Helper displaying confirmation message
+    * @param string $string
+    * @return string
+    */
```

Comment:
Sorry for being picky about this, but it seems a space is missing before the first `*`. They should be all aligned like for `displayError()`